BlueShift-iOS-SDK/BlueShift.m

```diff
@@ -7,17 +7,27 @@
 
 #import "BlueShift.h"
 #import <UserNotifications/UserNotifications.h>
-#import "InApps/BlueShiftInAppNotificationManager.h"
+#import "BlueShiftInAppNotificationManager.h"
 #import "BlueShiftNotificationConstants.h"
 #import "BlueshiftLog.h"
 #import "BlueshiftConstants.h"
+#import "BlueShiftInAppNotificationConstant.h"
+#import "BlueShiftInAppNotificationHelper.h"
 
-BlueShiftAppDelegate *_newDelegate;
 BlueShiftInAppNotificationManager *_inAppNotificationMananger;
 static BlueShift *_sharedBlueShiftInstance = nil;
 
 @implementation BlueShift
 
+static dispatch_queue_t bsft_serial_queue() {
```

Comment:
using snake case here is as per the conventions?

Reply:
Yes, for dispatch queues snake case convention is used in Objective c.